website/www/site/content/en/documentation/runtime/environments.md

```diff
@@ -17,147 +17,232 @@ limitations under the License.
-apache/beam_java8_sdk              latest              ...                 2 weeks ago       ...
-apache/beam_java11_sdk             latest              ...                 2 weeks ago       ...
-apache/beam_python2.7_sdk          latest              ...                 2 weeks ago       ...
-apache/beam_python3.5_sdk          latest              ...                 2 weeks ago       ...
-apache/beam_python3.6_sdk          latest              ...                 2 weeks ago       ...
-apache/beam_python3.7_sdk          latest              ...                 2 weeks ago       ...
-apache/beam_go_sdk                 latest              ...                 2 weeks ago       ...
+apache/beam_java8_sdk              latest              ...                 1 min ago         ...
+apache/beam_java11_sdk             latest              ...                 1 min ago         ...
+apache/beam_python3.6_sdk          latest              ...                 1 min ago         ...
+apache/beam_python3.7_sdk          latest              ...                 1 min ago         ...
+apache/beam_python3.8_sdk          latest              ...                 1 min ago         ...
+apache/beam_go_sdk                 latest              ...                 1 min ago         ...
 ```
 
-### Overriding default Docker targets
-
-The default [tag](https://docs.docker.com/engine/reference/commandline/tag/) is sdk_version defined at [gradle.properties](https://github.com/apache/beam/blob/master/gradle.properties) and the default repositories are in the Docker Hub `apache` namespace.
-The `docker` command-line tool implicitly [pushes container images](#pushing-container-images) to this location.
+If you did not provide a custom repo/tag as additional parameters (see below), you can retag the image and [push](https://docs.docker.com/engine/reference/commandline/push/) the image using Docker to a remote repository.
 
-To tag a local image, set the `docker-tag` option when building the container. The following command tags a Python SDK image with a date.
-```
-./gradlew :sdks:python:container:py36:docker -Pdocker-tag=2019-10-04
 ```
+export BEAM_SDK_VERSION="2.26.0"
+export IMAGE_NAME="myrepo/mybeamsdk"
+export TAG="${BEAM_SDK_VERSION}-custom"
 
-To change the repository, set the `docker-repository-root` option to a new location. The following command sets the `docker-repository-root`
-to a repository named `example-repo` on Docker Hub.
-```
-./gradlew :sdks:python:container:py36:docker -Pdocker-repository-root=example-repo
+docker tag apache/beam_python3.6_sdk "${IMAGE_NAME}:${TAG}"
+docker push "${IMAGE_NAME}:${TAG}"
 ```
 
-## Pushing container images
+**NOTE**: After pushing a container image, verify the remote image ID and digest matches the local image ID and digest output from `docker_images`
 
-After [building a container image](#building-container-images), you can store it in a remote Docker repository.
+##### Additional build parameters
```

Comment:
I think it's OK to keep this at H3 instead of H4

### Additional build parameters